packages/client-adapters/src/adapters/tokenization/adapter.js

```diff
@@ -47,16 +48,83 @@ class AlpheiosTokenizationAdapter extends BaseAdapter {
   }
 
   /**
-  * This method constructs full url for getting data
+  * This method uploads default config data from tokenization service
+  * @return {Array} - array of settings
+  */
+  async getConfig () {
+    try {
+      const url = this.createConfigFetchURL()
+      if (!url) {
+        this.addError(this.l10n.getMsg('TOKENIZATION_FETCH_OPTIONS_ERROR'))
+        return
+      }
+
+      if (this.sourceData) {
+        return this.sourceData
+      } else {
+        const configData = await this.fetch(url)
+
+        return this.formatSettings(configData)
+      }
+    } catch (error) {
+      this.addError(this.l10n.getMsg('TOKENIZATION_FETCH_ERROR', { message: error.message }))
+    }
+  }
+
+  /**
+   * Converts JSON response to Options for text and tei
+   * @param {Object} configData - Response from config fetch request
+   */
+  formatSettings (configData) {
+    return {
+      tei: this.convertToOptions(configData, 'tei'),
+      text: this.convertToOptions(configData, 'text')
+    }
+  }
+
+  /**
+   *
+   * @param {Object} configData Response from config fetch request
+   * @param {String} textType - tei/text
+   */
+  convertToOptions (configData, textType) {
+    const configDataPath = configData.paths[`/tokenize/${textType}`].post
+
+    const exludeParameters = ['lang', 'direction']

```

Comment:
why do you exclude these?

Reply:
Because it won't be defined through application settings.
It would be defined through settings inside each aligned request.

These are for defining setting in application options.